src/test/java/MapersTests/MapRaceTests.java

```diff
@@ -0,0 +1,35 @@
+    private final String badOutputMapper = "{\"ability_score\":{\"index\":\"dex\",\"name\":\"DEX\",\"url\":\"/api/ability-scores/dex\"},\"noslda\":2}";
+    private final MapRace testObject = new MapRace();
+
+    @Test
+    public void mapperHappyPath() {
+        String resultForString = testObject.mapRace(outputResponseBody).toString();
+        Assertions.assertEquals(outputMapper, resultForString);
+    }
+
+    @Test
+    public void mapperUNHappyPath() {
+        Assertions.assertNotEquals(outputMapper, " ");
```

Comment:
same thing for testing. This is not a strong check

Reply:
Corrected!